### DISCUSSION TITLE
Ordered List Small Icons

### DISCUSSION THREAD

**Discussion by mabourehab** [nvim-neorg/neorg#1158] [Q&A]
I have a question regarding the icons used by ordered lists.

Some icons will look extremely small, while others will have the correct size. Here's an example:

![image](https://github.com/nvim-neorg/neorg/assets/87973909/af8f445d-757c-4635-bc9c-c040771ac7e1)

I'm thinking this is likely an issue with my terminal settings, but can't determine what. I'm using Ubuntu 23.04 on WSL2 on Windows 11 with the Terminal app. 

Here are my terminal settings:
![image](https://github.com/nvim-neorg/neorg/assets/87973909/d47983bd-bcd6-43f6-8828-30efbdfcee63)
![image](https://github.com/nvim-neorg/neorg/assets/87973909/f6eae8f9-2c18-4008-ae97-70ef81e24d15)

Appreciate if any one has any ideas or has seen this before. Thanks!

**Comment by pysan3** [comment]
I'd recommend using the windows terminal preview (you can download from the app store) since the stable version does not know how to handle nerdfonts v3 yet.

Would you share what happens over on preview? If it still does not get fixed, maybe it's a font problem.

**Reply by mabourehab** [reply]
Hi! Thanks for the suggestion! Unfortunately, that did not cause the issue to get fixed.

Confirmation I'm using Terminal Preview now:
![image](https://github.com/nvim-neorg/neorg/assets/87973909/e3134d03-4f53-43bf-98ab-ec7ab42aad5f)


New file with how the lists look (Also, ran Neorg sync-parsers before trying):
![image](https://github.com/nvim-neorg/neorg/assets/87973909/8ef995b2-9edf-42f1-893a-5a7c1a4f130d)


The terminal settings I kept the same as what I posted in the original post.

I forgot to post this before, here is my init.vim:
```
$ cat $nvim
set number
set autoindent
set tabstop=4
set shiftwidth=4
set smarttab
set softtabstop=4
set mouse=a
hi Search cterm=none ctermfg=DarkMagenta ctermbg=White
set conceallevel=2
set concealcursor="nc"

call plug#begin()

Plug 'http://github.com/tpope/vim-surround' " Surrounding ysw)
Plug 'https://github.com/tpope/vim-repeat'
" Plug 'https://github.com/preservim/nerdtree' " NerdTree
Plug 'https://github.com/nvim-neo-tree/neo-tree.nvim'
Plug 'https://github.com/MunifTanjim/nui.nvim'
Plug 'https://github.com/tpope/vim-commentary' " For Commenting gcc & gc
Plug 'https://github.com/vim-airline/vim-airline' " Status bar
Plug 'https://github.com/ryanoasis/vim-devicons' " Developer Icons
Plug 'https://github.com/nvim-neorg/neorg'
Plug 'https://github.com/nvim-lua/plenary.nvim'
Plug 'https://github.com/nvim-treesitter/nvim-treesitter'
Plug 'https://github.com/rebelot/kanagawa.nvim'
Plug 'https://github.com/ellisonleao/gruvbox.nvim'
Plug 'https://github.com/Mofiqul/vscode.nvim'
Plug 'https://github.com/marko-cerovac/material.nvim'

set encoding=UTF-8

call plug#end()

nnoremap <C-q> :Neotree toggle<CR>
nnoremap <C-s> :Neotree dir=../<CR>

colorscheme material-palenight


lua << EOF
run = ":Neorg sync-parsers",
require('neorg').setup {
    load = {
        ["core.defaults"] = {}, -- Loads default behaviour
                ["core.keybinds"] = {
                        config = {
                                neorg_leader = ";" },
                },
        ["core.concealer"] = {
                        config = {
                                icon_preset = "diamond",
                                icons = {
                                        code_block = {
                                                conceal = true,
                        },
                                },
                        }, -- Adds pretty icons to your documents
                },
        ["core.dirman"] = { -- Manages Neorg workspaces
            config = {
                workspaces = {
                                        studies = "~/studies",
                },
            },
        },
                ["core.autocommands"] = {},
                ["core.integrations.treesitter"] = {},
                ["core.syntax"] = {},
                ["core.itero"] = {},
    },
}
EOF
```

**Comment by jototland** [comment]
I have the same problem. I want ordered list numbers to be formatted using regular ascii characters, even if the concealer is enabled. Is there any way to do that? 

I tried replacing the generator function for "ordered" in the config with the identity function, but neorg still prints these weird symbols instead of regular numbers. I'm stuck.

PS: I'm happy with my current terminal and font settings and do not want to change them.

**Reply by champignoom** [reply]
>  I want ordered list numbers to be formatted using regular ascii characters

nvim v0.10+ (dev) should behave like that by default.

**Reply by jototland** [reply]
Thanks!
